snappass/main.py

```diff
@@ -31,6 +31,8 @@ def set_password(password, ttl):
 
 def get_password(key):
     password = redis_client.get(key)
+    if password is not None:
```

Comment:
Not sure you saw https://github.com/pinterest/snappass/pull/19/commits/a46fc40aa321dda36a70a43b6a035aeb3bdb0eb9#diff-f4bc68ff01eef7bed2f3a64cad7d37caR26.

If so, what kind of additional testing are you thinking of? I wasn't exactly sure of the approach, since string handling is very different in python 2 and 3.